docs/root/configuration/listeners/network_filters/connection_limit_filter.rst

```diff
@@ -0,0 +1,106 @@
+.. _config_network_filters_connection_limit:
+
+Connection Limit Filter
+=======================
+
+Background
+----------
+
+Network connections are a limited resource that we need some functionality to protect.
+Envoy has the capability to limit the rate of new connections via the L4 `local rate limit filter <https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/network_filters/local_rate_limit_filter>`_.
+It would be useful to be able to limit the number of connections on a filter chain basis or based on some descriptors from the request.
```

Comment:
Can we rephrase this now that the feature is landing?

It would be -> It is  

etc?

and move the goals inline, etc.

Reply:
Updated the doc.